js/index.js

```diff
@@ -26,11 +26,16 @@ class OffsetHelperClient {
     constructor() {
         /**
          * @notice to be used on the backend
```

Comment:
Why do we need the `js/` directory? It looks like we only need to maintain `ts/`, right?

Reply:
Wouldn't it be useful for js (non-ts) projects outside of Toucan?

Reply:
Need to think about this more.

Reply:
So the JS code is generated by the build process, we can ignore it in git and only include it in the published module, right? From my experience, this is what other npm modules do.